Services/Dashboard/classes/class.ilDashboardGUI.php

```diff
@@ -219,14 +213,17 @@ public function executeCommand(): void
                 $this->ctrl->forwardCommand($ggui);
                 $this->tpl->printToStdout();
                 break;
-
-            case "ildashboardrecommendedcontentgui":
-                $gui = new ilDashboardRecommendedContentGUI();
-                $this->ctrl->forwardCommand($gui);
-                break;
-            case "ilstudyprogrammedashboardviewgui":
-                $gui = new ilStudyProgrammeDashboardViewGUI();
-                $this->ctrl->forwardCommand($gui);
+            case strtolower(ilLearningSequenceBlockGUI::class):
+            case strtolower(ilMembershipBlockGUI::class):
+            case strtolower(ilRecommendedContentBlockGUI::class):
+            case strtolower(ilSelectedItemsBlockGUI::class):
+            case strtolower(ilStudyProgrammeBlockGUI::class):
+                $gui = new $next_class();
+                $ret = $this->ctrl->forwardCommand($gui);
+                if ($ret != "") {
```

Comment:
Please use either `!$ret` or `$ret !== ""` depending on the fact that you know the type of `$ret` or not.